packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_badgebutton.scss

```diff
@@ -11,24 +11,24 @@
    Widget styles
 ========================================================================== */
 .widget-badge-button {
-  display: inline-block;
+    display: inline-block;
```

Comment:
Looks to me these tabulation changes are by mistake ?

Reply:
We've implemented pretty-quick for sass now, so this will happen in some cases (I think)